bin/katello-certs-check

```diff
@@ -112,6 +112,24 @@ function check-ca-bundle () {
     fi
 }
 
+function check-cert-encoding () {
+    printf "Checking server certificate's encoding: "
+    openssl x509 -inform per -in $CERT_FILE -noout -text &> /dev/null
+    if [ $? == "0" ]; then
+        success
+    else
+        openssl x509 -inform der -in $CERT_FILE -noout -text &> /dev/null
+        if [ $? == "0" ]; then
+            error 8 "The server certificate is in DER encoding, which is not
+            compatible. Convert the certificate to PEM encoding,
+            then test it again."
```

Comment:
Thanks, @sean797. I originally had the `openssl` command included in the output, but it was being run, rather than simply being displayed. At least that was my theory at the time. I will try to add this, as it's much more user-friendly to provide the exact command, instead of stating "Fix it".